Changes.md

```diff
@@ -1,3 +1,11 @@
+0.59.x.x (relative to 0.59.9.1)
+========
+
+Features
+-----
```

Comment:
The `0.59` series is in bugfix only status now, so we'll need to rebase this PR against `0.60_maintenance`.